doc/release-notes.md

```diff
@@ -10,6 +10,43 @@ Option handling
 - The `-reindex` and `-reindex-chainstate` options now imply `-rescan`
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
+- A new string-valued option, `-allowdeprecated` has been introduced to allow
```

Comment:
This should be its own heading under `Notable changes` (maybe "Feature deprecation and removal"), not a subsection of `Option handling`.